packages/autorag/frontend/src/app/components/run-results/PatternDetailsModal/PatternDetailsModal.scss

```
@@ -1,10 +1,139 @@
 .autorag-pattern-details-modal {
   --pf-v6-c-modal-box--MaxWidth: 1300px;
   --pf-v6-c-modal-box--m-lg--lg--MaxWidth: 1300px;
+  --autorag-modal--body--PaddingLeft: var(--pf-t--global--spacer--lg);
   height: 80vh;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)